docker_run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+###  SET HOST FILE PATHS HERE. All paths must be absolute!  ###
+HOST_CFG=$(pwd)/rabbitmq-worker.conf
+HOST_LOG=$(pwd)/rabbitmq-worker.log
+HOST_ERR=$(pwd)/rabbitmq-worker.err
```

Comment:
It's more of a proper docker configuration then. It allows you to run `docker logs` to see the output. From my understanding, you also want to do this because it fixes the issues with log rotation, and also reduces complexity.

This also allows you to use the different docker log drivers. https://docs.docker.com/engine/admin/logging/overview/

Overall, it's more dockerish.